torch/_inductor/cudagraph_trees.py

```diff
@@ -416,25 +417,43 @@ def cudagraphify(
     constants: Tuple[torch.Tensor, ...] = (),
     placeholders: Tuple[PlaceholderInfo, ...] = (),
     mutated_input_idxs: Tuple[int, ...] = (),
+    compile_id: Optional[CompileId] = None,
 ) -> Tuple[ModelType, OutputType]:
-    manager = get_container(device_index).get_tree_manager()
     assert not (is_backward and is_inference)
     mode = (
         CompilationMode.BACKWARD
         if is_backward
         else (CompilationMode.INFERENCE if is_inference else CompilationMode.FORWARD)
     )
 
-    return manager.add_function(
-        model,
-        inputs,
-        static_input_idxs,
-        stack_traces,
-        mode,
-        constants,
-        placeholders,
-        mutated_input_idxs,
-    )
+    with dynamo_timed(
+        "cudagraphify.get_container",
+        log_pt2_compile_event=True,
+        # This operation is strictly cudagraph overhead, so we count it
```

Comment:
This part is important. @ezyang, can you vet the rationale in the summary? I talked to @eellison and I'm trying to capture only the _additional_ overhead of cudagraphs, e.g., the call to `run_eager` elsewhere does not get added to the column `runtime_cudagraphify_time_us`. Check out the perfetto traces linked in the summary to see what I mean.

Also something to note: experimentally, the overhead you'll see in the sample perfetto traces is coming from here: https://github.com/pytorch/pytorch/blob/bb574abe73aa84fd9b895b4c7c39ad79bd53f037/torch/_inductor/cudagraph_trees.py#L1874
We could wrap the dynamo_timed around that specifically, but it'd involve passing the compile_id through a bunch of stuff and capturing the timing here seems almost as good.